File: GTTS/INL.PersonService.Models/SavePersonAttachment_Param.cs

namespace INL.PersonService.Models
{
    public class SavePersonAttachment_Param : ISavePersonAttachment_Param
    {
        public long PersonID { get; set; }
        public long? FileID { get; set; }
        public string FileName { get; set; }
        public int? FileVersion { get; set; }
        public string PersonAttachmentType { get; set; }
        public string Description { get; set; }
        public bool? IsDeleted { get; set; }
        public int ModifiedByAppUserID { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.PersonService.Models/save-person-attachment_param.ts


import { ISavePersonAttachment_Param } from './isave-person-attachment_param';

export class SavePersonAttachment_Param implements ISavePersonAttachment_Param {
  
	public PersonID: number = 0;
	public FileID?: number;
	public FileName: string = "";
	public FileVersion?: number;
	public PersonAttachmentType: string = "";
	public Description: string = "";
	public IsDeleted?: boolean;
  
}


